source/community.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: has_complimentary
+title: '#teamSass'
+introduction: >
+  Sass has an awesome community of designers and developers who love to spread
+  the word and help people out. Here we&rsquo;ve collected some resources.
+  Happy&nbsp;Styling!
```

Comment:
We could run this through `markdown` or `typogr` to do this automatically instead of manually.